integration/templates/astro-node/src/pages/pricing-table.astro

```diff
@@ -0,0 +1,10 @@
+---
+import { Experimental_PricingTable } from "@clerk/astro/components";
+import Layout from "../layouts/Layout.astro";
+---
+
+<Layout title="Welcome to Astro.">
+  <div class="w-full flex justify-center">
+    <Experimental_PricingTable />
```

Comment:
The React/Next.js components are named `__experimental_PricingTable`. Why not the Astro / Astro & React one?

Reply:
`.astro` and `.vue` files does not like `__experimental_PricingTable` 😞